src/status_im2/contexts/wallet/events.cljs

```diff
@@ -10,3 +56,74 @@
   {:events [:wallet/clean-scanned-address]}
   [{:keys [db]}]
   {:db (dissoc db :wallet/scanned-address)})
+
+(defn fetch-address-suggestions
+  [{:keys [db]} [address]]
+  {:db (assoc db
+              :wallet/local-suggestions
+              (cond
+                (= address
+                   (get-in
+                    address-local-suggestion-saved-contact-address-mock
```

Comment:
perhaps it's also okay to fill the db with mock data on login or something to that effect so we have one place with it all? 🤔

Reply:
I am not sure, it will be easier to replace the logic in the events later than add more unnecessary logic in the UI which is just subscribing to events / keys.

Reply:
> perhaps it's also okay to fill the db with mock data or something to that effect so we have one place with it all? 🤔

That's a good idea, but maybe is better to do it after we decide the structure of app-db? wdyt?

Reply:
sure, seems fine. Perhaps a reasonable interim is to put this internal logic in a function with some name such as `mock-*`  and define it in the temp folder so that it is easy to track these mock pieces - i.e we know all references to `temp` ns have some mocked behaviour. 
Whatever way you decide to handle is good by me, just giving my thoughts 👍